price_feeds/solana/send_usd/program/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "send-usd"
+version = "0.1.0"
+description = "Created with Anchor"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "lib"]
+name = "send_usd"
+
+[features]
+no-entrypoint = []
+no-idl = []
+no-log-ix-name = []
+cpi = ["no-entrypoint"]
+default = []
+
+[dependencies]
+anchor-lang = "0.30.0"
+solana-program = "1.17.3"
```

Comment:
bump this due to getting `ProgramError::ArithmeticOverflow` when compiling the program, more info here: https://github.com/coral-xyz/anchor/pull/2975

we are using pyth-solana-receiver-sdk which uses anchor-lang 0.30.0 whereas the package here intially used 0.28.0 which caused this issue so bumping both to the working version